engines/bastion_katello/app/assets/stylesheets/bastion_katello/bastion_katello.scss

```diff
@@ -26,4 +26,7 @@
     text-overflow: ellipsis;
   }
 
+  .breadcrumb {
+    margin-bottom: 0px;
```

Comment:
My point here is that we don't want to create a bunch of application specific styles when we should be adhering to the [recommendations set forth by patternfly](https://www.patternfly.org/pattern-library/widgets/#breadcrumbs).  In this case patternfly doesn't (or hasn't yet) touched on breadcrumbs so I don't think we should either.

Ultimately gaining vertical space in katello needs to be viewed holistically and also with foreman in mind.  We could, for instance, gain an easy 24px by removing (i.e. combining with the bottom header) the header with the hardhat in it.